geocoder/src/main/scala/grasshopper/geocoder/search/census/AddressInterpolator.scala

```diff
@@ -89,8 +89,9 @@ object AddressInterpolator {
     } else if (!addressIsEven && !leftRangeIsEven) {
       "L"
     } else {
-      log.warn(s"Could not determine if range is even in feature: ${f.toString}")
-      "R"
+      val default = "R"
```

Comment:
I think its a little more explicit as it is, but yes, could have done that and saved a line of code.